src/pages/user.tsx

```diff
@@ -42,32 +81,59 @@ const userData = [
 export function User() {
   const [showModal, setShowModal] = useState(false);
   const [data, setData] = useState<UserInterface[]>(userData);
+  const [status, setStatus] = useState("All");
 
   const openModal = () => {
     setShowModal(true);
   };
 
   const handleSearchTermChange = (term: string) => {
```

Comment:
Change the term type to Status type created in the other file.

Reply:
i changed to userStatus because its more particular to the user interface and its more easy to understand